KicadModTree/nodes/specialized/PadArray.py

```diff
@@ -106,8 +109,13 @@ def _initPincount(self, **kwargs):
             raise ValueError('{pc} is an invalid value for pincount'.format(pc=self.pincount))
 
         self.exclude_pin_list = []
-        if kwargs.get('exclude_pin_list'):
-            self.exclude_pin_list = kwargs.get('exclude_pin_list')
+        if kwargs.get('hidden_pins') and kwargs.get('hidden_pins'):
```

Comment:
That should be `if kwargs.get('hidden_pins') and kwargs.get('deleted_pins'):`

Reply:
Good catch. Fixed.